### DISCUSSION TITLE
Output attachment asset URL rather than CP URL in email

### DISCUSSION THREAD

**Discussion by zizther** [verbb/formie#1572] [General] [answered]
I have an email notification which is sent to the user on submission. I am sending them a copy of the email, which includes the attachments.
When using the `All Form Fields` variable it shows the link to the asset but links to the asset within the control panel, not a publicly accessible link where the asset resides. In my instance users don't have accounts or need to access the control panel, and I don't really want to give away the CP location either.

Is it possible to output the asset volume URL rather than the CP URL?

**Comment by engram-design** [comment, marked as answer]
This is on our todo list to implement https://github.com/verbb/formie/issues/753

**Reply by zizther** [reply]
Thanks for the heads up, would be good to see this functionality in 👍
